testsuite/manualmode/pom.xml

```diff
@@ -632,6 +633,8 @@
                                         <exclude>org.wildfly.core.test.standalone.mgmt.events.*TestCase</exclude>
                                         <!-- Requires modification of the process-uuid file inside the Jar -->
                                         <exclude>org.jboss.as.test.manualmode.logging.ProcessUuidTestCase</exclude>
+                                        <!-- Validates content of traditional unzipped installation. (Works, but is irrelevant to bootable jar testing.) -->
+                                        <exclude>org.jboss.as.test.manualmode.provisioning.ProvisioningConsistencyTestCase</exclude>
```

Comment:
This looks unrelated to these changes, isn't it? I am not against it, just trying to link the points.

Reply:
Yes, it's unrelated. I have a bunch of branches in this general 'test provisioning' topic area and I got tired of switching back and forth. :)

Reply:
Ok, thank you for confirming @bstansberry , I also wanted to be sure it was not unintentional